packages/gatsby-remark-prismjs/README.md

```diff
@@ -135,7 +135,7 @@ CSS along your PrismJS theme and the styles for `.gatsby-highlight-code-line`:
 
 If you want to add line numbering alongside your code, you need to
 import the corresponding CSS file from PrismJS, right after importing your
-colorscheme in `layout/index.js`:
+colorscheme in `src/components/layout.js`:
 
 ```javascript
 // layouts/index.js
```

Comment:
Don’t forget this comment 😛

Reply:
Sorry, this is my first contribution on this project and I see that you have collaborated before. Is there anything wrong with my PR? Thank beforehand for any feedback.

Reply:
Ah, no, your proposed change is fine I guess. Just wanted to mention that you also should change the comment inside the codeblocn, too

Reply:
@albertod Do you have time to make that change?

Reply:
Done 👍